cmd/snap-seccomp/main_test.go

```diff
@@ -0,0 +1,404 @@
+		{"setns - CLONE_NEWNETT", `cannot parse line: cannot parse token "CLONE_NEWNETT" \(line "setns - CLONE_NEWNETT"\)`},
+		{"setns - CL0NE_NEWNET", `cannot parse line: cannot parse token "CL0NE_NEWNET" \(line "setns - CL0NE_NEWNET"\)`},
+		// test_bad_seccomp_filter_args_mknod (various typos in input)
+		{"mknod - |S_IFIF", `cannot parse line: cannot parse token "S_IFIF" \(line "mknod - |S_IFIF"\)`},
+		{"mknod - |S_IFIFOO", `cannot parse line: cannot parse token "S_IFIFOO" \(line "mknod - |S_IFIFOO"\)`},
+		{"mknod - |S_!FIFO", `cannot parse line: cannot parse token "S_IFIFO" \(line "mknod - |S_!FIFO"\)`},
+		// test_bad_seccomp_filter_args_null
+		{"socket S\x00CK_STREAM", `cannot parse line: cannot parse token .*`},
+		{"socket SOCK_STREAM\x00bad stuff", `cannot parse line: cannot parse token .*`},
+
+		// test_bad_seccomp_filter_args
+		{"mbind - - - - - - 7", `cannot parse line: too many arguments specified for syscall 'mbind' in line.*`},
+		{"mbind 1 2 3 4 5 6 7", `cannot parse line: too many arguments specified for syscall 'mbind' in line.*`},
+		// test_bad_seccomp_filter_args_prctl
+		{"prctl PR_GET_SECCOM", `cannot parse line: cannot parse token "PR_GET_SECCOM" .*`},
+		{"prctl PR_GET_SECCOMPP", `cannot parse line: cannot parse token "PR_GET_SECCOMPP" .*`},
+		{"prctl PR_GET_SECC0MP", `cannot parse line: cannot parse token "PR_GET_SECC0MP" .*`},
+		{"prctl PR_CAP_AMBIENT_RAIS", `cannot parse line: cannot parse token "PR_CAP_AMBIENT_RAIS" .*`},
+		{"prctl PR_CAP_AMBIENT_RAISEE", `cannot parse line: cannot parse token "PR_CAP_AMBIENT_RAISEE" .*`},
+		// test_bad_seccomp_filter_args_prio
+		{"setpriority PRIO_PROCES 0 >=0", `cannot parse line: cannot parse token "PRIO_PROCES" .*`},
+		{"setpriority PRIO_PROCESSS 0 >=0", `cannot parse line: cannot parse token "PRIO_PROCESSS" .*`},
+		{"setpriority PRIO_PR0CESS 0 >=0", `cannot parse line: cannot parse token "PRIO_PR0CESS" .*`},
+		// test_bad_seccomp_filter_args_quotactl
+		{"quotactl Q_GETQUOT", `cannot parse line: cannot parse token "Q_GETQUOT" .*`},
+		{"quotactl Q_GETQUOTAA", `cannot parse line: cannot parse token "Q_GETQUOTAA" .*`},
+		{"quotactl Q_GETQU0TA", `cannot parse line: cannot parse token "Q_GETQU0TA" .*`},
+		// test_bad_seccomp_filter_args_socket
+		{"socket AF_UNI", `cannot parse line: cannot parse token "AF_UNI" .*`},
+		{"socket AF_UNIXX", `cannot parse line: cannot parse token "AF_UNIXX" .*`},
+		{"socket AF_UN!X", `cannot parse line: cannot parse token "AF_UN!X" .*`},
+		{"socket - SOCK_STREA", `cannot parse line: cannot parse token "SOCK_STREA" .*`},
+		{"socket - SOCK_STREAMM", `cannot parse line: cannot parse token "SOCK_STREAMM" .*`},
+		// test_bad_seccomp_filter_args_termios
+		{"ioctl - TIOCST", `cannot parse line: cannot parse token "TIOCST" .*`},
+		{"ioctl - TIOCSTII", `cannot parse line: cannot parse token "TIOCSTII" .*`},
+		{"ioctl - TIOCST1", `cannot parse line: cannot parse token "TIOCST1" .*`},
```

Comment:
Correct, there is no size limit right now for the syscall length. I can add that in a followup, I agree that its not that critical.